### PR TITLE
#14 add template, improved doc, add examples

### PR DESCRIPTION
**Pull request by MV88**
issue #14

add template to config options with a default value
improved doc in general 

added an example on how to configure configOptions.json

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Description
+<!-- A few sentences describing the overall goals of the pull request' s commits. -->
+
+**Please check if the PR fulfills these requirements**
+- [ ] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
```
Comment (mbarto) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: This project should not be tied to MapStore2, also for the example I would use a more generic template (I would look if anyone created some generic templates to inherit)

File: createPR.js [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
 const fs = require("fs");
 
 const issue_number = process.argv[2];
 
+// the following resource will be applied to the cwd (current working directory)
 const git = simpleGit(".");
-const configFile = fs.readFileSync("./hackattoni.json", "utf8");
-const {github_token, remote, fork} = JSON.parse(configFile);
-const octokit = new Octokit({ auth: github_token });
+const configOptions = fs.readFileSync("./configOptions.json", "utf8");
```
Comment (mbarto) [inline createPR.js]: I think configOptions.json is not very meaningful as a name, it should have a name that reminds its purpose. Remember that you should put this file in your project repo. Think of other configuration files for tools, e.g. babel.config.json.

**Review by mbarto — CHANGES_REQUESTED** [review]

File: createPR.js (hunk 2 above)
Comment (MV88) [reply createPR.js]: right and i spent 10 minutes thinking, rethinking and renaming it :)
alternatives
- [ ] createPROptions.json
- [ ] create_PR_Options.json
- [ ] automatic_create_PR_options.json
- [ ] hackattoni.json (didn't like much this)
- other:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (MV88) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: sounds good
what about this one?
https://github.com/stevemao/github-issue-templates/blob/master/questions-answers/PULL_REQUEST_TEMPLATE.md

an alternative would be to remove the link to the guidelines and leave the rest

File: createPR.js (hunk 2 above)
Comment (MV88) [reply createPR.js]: what about the name?

Comment (mbarto) [reply createPR.js]: Thinking out loud: maybe we should just choose a name for this project and then use <projectName>.json

Some project names coming to my mind:
 - [ ] git-workflow-tools
 - [x] automata
 - [ ] just-do-it

**Pull request merged**